.eslintrc.yml

```diff
@@ -0,0 +1,15 @@
+env:
+  node: true
+  es2020: true
+extends: 'eslint:recommended'
+parserOptions:
+  ecmaVersion: 11
+rules: 
```

Comment:
If there is a const vs let rule somewhere, that would be tight. Also bitchy if anyone tries to write anonymous functions without arrow syntax